flink-python/pyflink/table/tests/test_table_environment_api.py

```diff
@@ -296,6 +296,15 @@ def test_create_table_environment_with_blink_planner(self):
             planner.getClass().getName(),
             "org.apache.flink.table.planner.delegation.StreamPlanner")
 
+        t_env = StreamTableEnvironment.create(
```

Comment:
Would be great if we can also add tests for flink planner. FYI: test will be failed for flink planner with the current PR.